trunner/testcase.py

```diff
@@ -89,7 +89,8 @@ def sysexec(self, proc):
     def exec_test(self, proc):
         try:
             # Wait for a prompt
-            proc.expect_exact('(psh)% ')
+            # On armv7a9-zynq7000-qemu jffs initialization may take to ~20s, that's why it's set to 25
+            proc.expect_exact('(psh)% ', timeout=25)
```

Comment:
If this currently happen only on `armv7a9-zynq7000-qemu` maybe increase timeout only for this target? under self there should be a target.
```suggestion
            # comment why we need longer wait for armv7a9-zynq7000-qemu
            prompt_timeout = 25 if self.target == "armv7a9-zynq7000-qemu" else 8
            proc.expect_exact('(psh)% ', timeout=prompt_timeout)
```
This may need additional parametrization in the future as there may be a need for different waiting times for different targets.

Reply:
Thanks, I think it may be a better solution.